### ISSUE TITLE
save docx file takes too long

### ISSUE THREAD

**Issue by fzargarbashi** [ONLYOFFICE/DesktopEditors#1729]
### This issue is unique.

- [X] I have used the [search tool](https://github.com/ONLYOFFICE/DesktopEditors/issues?q=) and did not find an issue describing my bug.

### Operating System

Linux (DEB package)

### Version information

version 8.2.1.38 (snap)

### Expected Behavior

After editing a small docx document (~600 kb) with track changes on and clicking on save. The 'saving' window appears and it saves the file in place within a few seconds. 

### Actual Behavior

The 'saving' window appears but it stucks there for more than 30 minutes. The app is not freezed as you can click the onlyoffice menu but the you can not navigate the saving document.

**Comment by TatianaKirillova** [comment]
Hello, @fzargarbashi ! I can't reproduce your problem yet. Please specify your OS and, if possible, attach the problematic file.

**Comment by thisisartman** [comment]
While not this long, I have experienced long load and save times for all files... PDF, Docx, ODF, XLSX & ODS. Interestingly, the Open Document formats take just as long as the proprietary formats to open and save, something I assumed would not be an issue.

**Comment by TatianaKirillova** [comment]
@thisisartman It's hard for me to say anything without specific files

**Comment by thisisartman** [comment]
@TatianaKirillova if you want, I could share a screen-recording as well as the file...

**Comment by TatianaKirillova** [comment]
@thisisartman Yes, that would clarify the situation a lot! Thank you!

**Comment by thisisartman** [comment]
Here you go... This is the video demonstrating the delays for _xlsx_ or _ods_ file:
https://github.com/user-attachments/assets/3a10c1c9-6ece-414e-b013-537bea4d0bab

and the _docx_ file issue too:
https://github.com/user-attachments/assets/8a1e78c5-7163-4d25-be2c-92561491fb06

For reference, these are my device specs:
Acer Predator Triton 300
i7 9th Gen,
16GB RAM,
GTX 1650 4GB VRAM,
OS (Win 11, latest security update, avoiding feature updates for 6mths for stability) on 512GB SSD,
All software installed on SSD in C drive.

And here's the files:
[New Microsoft Word Document.docx](https://github.com/user-attachments/files/18183198/New.Microsoft.Word.Document.docx)
[Japan MBA plan.ods](https://github.com/user-attachments/files/18183202/Japan.MBA.plan.ods)

**Comment by TatianaKirillova** [comment]
@thisisartman I'm very sorry, but I can't seem to reproduce your problem. Opening from a local folder works in 2-3 seconds

**Comment by thisisartman** [comment]
@TatianaKirillova Hi... I don't know why its behaving this way on my system... Here's another video fo me trying to open a blank docx file.

https://github.com/user-attachments/assets/d0845cf5-2fa5-4c6c-a4f4-69dcdbe79f52

I don't know what else I could possibly do to make this work. Is there any common solution like a particular type of installation?

**Comment by TatianaKirillova** [comment]
@thisisartman I also try it on Win 11, is this behavior (creating an empty document) every time or just the very first?

**Comment by thisisartman** [comment]
@TatianaKirillova yes. Its every time... I am also on Windows 11...

**Comment by TatianaKirillova** [comment]
@thisisartman please record another video from the task manager when opening the file on the performance page

**Comment by thisisartman** [comment]
here you go... As you can see, 
https://github.com/user-attachments/assets/e0a46b5a-b61f-43ef-afa1-5b9a799b0671

Here's the processes before running OnlyOffice.
![Image](https://github.com/user-attachments/assets/59c2198b-bfc7-49d5-b977-fbf3802f8e68)

Hope that helps.

**Comment by LuckySpecialist** [comment]
HI @thisisartman, judging by the attached video, your system is very loaded at the time of launching the application and opening the document, in terms of RAM and CPU. For example, slow rendering of windows and mouse cursor, hard disk and wireless network activity.
Can you separate the problems, suspend all third-party processes that consume resources, and check whether the application is running and opening a file on the system without load?

**Comment by thisisartman** [comment]
>slow rendering of windows and mouse cursor

Yeah that's actually just the ShareX screen recording. I'll take one with OBS or the Windows recorder to get a higher quality one... 

> Can you separate the problems, suspend all third-party processes that consume resources, and check whether the application is running and opening a file on the system without load?

Absolutely. Also, Libre Office opens up files faster in similar situations. I'll try to post a comparison.

**Comment by alexonlyoffice** [comment]
@thisisartman, please specify if the issue is still relevant.  

If it still exists, please specify the following: 

1. What security systems and antiviruses are installed on the PC? If possible, please try to disable them and check again the time of the file saving; 
2. On the last video you showed the common loading of the system, if possible please make a video showing the load of our application;
3. What energy power profile is set during the tests? 

Please download this tool , it can ccollect process logs to show us more information about what is hapenning. 

1. Reproduce the situation
2. Run Procmon
3. Set up app filter to see the editor processes
4. Save the log as CSV

**Comment by Rita-Bubnova** [comment]
This issue was closed due to no response.
